<ui version="4.0" >
 <class>form_MainWindow</class>
 <widget class="QMainWindow" name="form_MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>I2PChat</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resourcen.qrc" >:/icons/userblue24.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="0" >
     <widget class="QComboBox" name="comboBox" >
      <property name="insertPolicy" >
       <enum>QComboBox::NoInsert</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QListWidget" name="listWidget" >
      <property name="contextMenuPolicy" >
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="minimumSize" >
    <size>
     <width>21</width>
     <height>30</height>
    </size>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resourcen.qrc" />
 </resources>
 <connections/>
</ui>
